src/homelist.js

```diff
@@ -62,10 +63,12 @@ const homeList = async (data) => {
     likesCount(likeBtn, likesArray, numOfLikes);
 
     likeBtn.addEventListener('click', async (e) => {
-      await postLike(e.target.id);
+      await postLike(likeBtn.id);
+
       const newLikes = await getLikes();
-      
-      likesCount(e.target, newLikes, numOfLikes);
+
```

Comment:
[OPTIONAL]
Consider changing the color of your like icon on press, I t could make your project look even better.